test/system/configuration_singleton_test.rb

```diff
@@ -41,10 +41,6 @@ def config_via_runner(env: 'development', envvars: '')
     Marshal.load(runner(code, env: env, envvars: envvars))
   end
 
-  test "tests should not be run with .env.local* files in directory" do
-    assert Dir.glob(".env.local{.development,.production,}").none?, "these tests should not be run with a .env.local or .env.local.development or .env.local.production"
-  end
-
```

Comment:
Why remove this? This is valuable because sometimes when I run tests I don't realize I have .env.local files modifying the environment that the tests run in.

Reply:
Because of...

https://github.com/OSC/ood-dashboard/blob/bb1f18d42d64b0d587deaa068ec0f06baba0aee7/config/configuration_singleton.rb#L143-L148

it will never run `.env.local`, `.env.development.local`, or `.env.production.local` in the test environment.

I also always have `.env.local` in my sandbox, and it is getting annoying to back it up, run tests, then move back.

Reply:
Great!